data/kernels/filmic.cl

```diff
@@ -639,15 +639,8 @@ static inline float4 filmic_split_v4(const float4 i,
   // Get final Ych in Kirk/Filmlight Yrg
   float4 Ych_final = pipe_RGB_to_Ych(o, matrix_in);
```

Comment:
looks like on the first loop iteration o is not set. code is suspicious....

Reply:
No problem. Actually I think that part of the code could be cleared up a bit. It is not 100% certain that casting a float4 to float* results in well defined behaviour.

Reply:
It's cleaned a bit now.